src/rules/own-methods-must-be-private.ts

```diff
@@ -1,16 +1,22 @@
 import { Rule } from 'eslint';
 import ts from 'typescript';
-import { isPrivate, stencilComponentContext, stencilDecorators, stencilLifecycle } from '../utils';
+import {
+  getDecoratorList,
+  isPrivate,
+  stencilComponentContext,
+  stencilDecorators,
+  stencilLifecycle,
+} from "../utils";
 
 const rule: Rule.RuleModule = {
   meta: {
     docs: {
-      description: 'This rule catches own class methods marked as public.',
-      category: 'Possible Errors',
-      recommended: true
+      description: "This rule catches own class methods marked as public.",
```

Comment:
I never ended up getting Prettier configured for this project. It might make sense to add it in a separate PR. If you'd like an example as to how we've done it in the past, feel free to take a look at https://github.com/ionic-team/create-stencil/commit/2d093d7a48861d07ccb5b683e0cd88e4a8d5dfa9 (although using Ionic's setup isn't a requirement here)

Reply:
Sure! Thanks!